cve_bin_tool/output_engine/print_mode.py

```diff
@@ -4,6 +4,7 @@
 from datetime import datetime
 from pathlib import Path
 from typing import Dict, Optional, Union
+from pathlib import Path
```

Comment:
```suggestion
from pathlib import Path
from typing import Dict, Optional, Union
```

Rearranging for the isort linter.